### PR TITLE
Adds detection of RSSRadio podcast client and RSSRadio Bot

### PR DESCRIPTION
**Pull request by poschi3**
I have RSSRadio in my podcast website logs.

As a [iOS only client](http://rssrad.io/#/rssradio):
* `RSSRadio/9220 (iPad;iOS;12.3.1)`
* `RSSRadio/9252`

As a Bot
* `RSSRadio (Push Notification Scanner;support@dorada.co.uk)`

### CODE REVIEW

File: regexes/oss.yml [hunk 1]
```diff
@@ -425,6 +425,9 @@
   name: 'iOS'
   version: ''
 
+- regex: 'RSSRadio/'
+  name: 'iOS'
+  version: '$1'
```
Comment (sgiehl) [inline regexes/oss.yml]: This one could be merged with the one 2 above: `'Podcasts/(?:[\d\.]+)|Instacast(?:HD)?/(?:\d\.[\d\.abc]+)|Pocket Casts, iOS|Overcast|Castro|Podcat|i[cC]atcher'`. Also the `$1` would be useless here, as the regex can't match anything

Comment (poschi3) [reply regexes/oss.yml]: Good point. I'll try it. Should I rebase and ammend my commits?

Comment (sgiehl) [reply regexes/oss.yml]: no need to rebase or amend. will be squashed when merging

**Comment by poschi3** [comment]
I think the PR is ready again :-)

**Review by sgiehl — APPROVED** [review]

**Pull request merged**